### PR TITLE
[dev-task] Reduce length of upload slack message

### PR DESCRIPTION
**Pull request by kobymeir**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:https://jira-dc.paloaltonetworks.com/browse/CIAC-205

### CODE REVIEW

File: Tests/scripts/gitlab_slack_notifier.py [hunk 1]
```diff
@@ -284,44 +290,78 @@ def unit_tests_results() -> list[dict[str, Any]]:
...
+            'title': f'Successfully uploaded {len(successful_packs)} Pack(s) to {marketplace_name}',
+            'color': 'good',
+        })
+        threaded_messages.append({
+            'fallback': f'Successfully uploaded {marketplace_name} Pack(s): '
+                        f'{", ".join(sorted({*successful_packs},key=lambda s: s.lower()))} to {marketplace_name}',
+            'title': f'Successfully uploaded {len(successful_packs)} Pack(s) to {marketplace_name}.',
+            'color': 'good',
+            'fields': [{
+                'title': 'Pack(s):',
```
Comment (yaakovpraisler) [inline Tests/scripts/gitlab_slack_notifier.py:314]: Not sure if this title is relevant, WDYT?

File: Tests/scripts/gitlab_slack_notifier.py [hunk 2]
```diff
@@ -406,14 +447,13 @@ def construct_slack_msg(triggering_workflow: str,
         # No color is needed in case of success, as it's controlled by the color of the test failures' indicator.
 
     title += title_append
-    title += f' - @{CI_SERVER_HOST}' if CI_SERVER_HOST else ''
```
Comment (yaakovpraisler) [inline Tests/scripts/gitlab_slack_notifier.py:409]: Why did you remove this?

**Review by yaakovpraisler — APPROVED** [review]
Looks good!
Just a few small comments -

File: Tests/scripts/gitlab_slack_notifier.py (hunk 1 above)
Comment (kobymeir) [reply Tests/scripts/gitlab_slack_notifier.py:314]: i removed the title. but i will keep the title empty, as it's mandatory field.

File: Tests/scripts/gitlab_slack_notifier.py (hunk 2 above)
Comment (kobymeir) [reply Tests/scripts/gitlab_slack_notifier.py:409]: the XDR migration is over, no longer needed.
